### DISCUSSION TITLE
How to add my GitHub repository in Google

### DISCUSSION THREAD

**Discussion by zs-3** [community/community#131915] [Programming Help] [answered]
### Select Topic Area

Question

### Body

It has been 3 days my GitHub repository is not coming in Google.
Please can anyone add it or tell me how to add.
Link: https://github.com/zs-3/ZS

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @zs-3 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Aditijainnn** [comment, marked as answer]
As long as your repository is set to  Public it's just a matter of time. Google web crawlers take time to scour the Internet for new webpages, but if they can get to it, it will get listed as soon as possible.

To improve how the repo is found, don't forget to add as much information to the repo as possible, such as a description, tags, and a detailed Readme.

Hopefully this helps!
